.changeset/old-coins-kneel.md

```diff
@@ -0,0 +1,5 @@
+---
+'xstate': minor
+---
+
+Added support for the `pure()` action creator.
```

Comment:
Would be great to include a short description of what it is good for - it's really valuable piece of information for a changelog, shortens the learning loop (one doesnt have to jump into the docs just to get a general sense of what it is useful for).